### ISSUE TITLE
Set timeout for open

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#10]
```
Operating system : Windows 7 64bit
.Net Framework version :
Office Version : 12
SeleniumWrapper version : 1.0.6.1

This there any possibility to set the timeout for open? It defaults to 30 
seconds but thats not enough for me in some cases. I read about 

selenium.setTimeout "40000"

but that doesn't work.

Thanks!

```

Original issue reported on code.google.com by `gruberal...@gmail.com` on 16 Oct 2012 at 1:24

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. I will add the selenium.setTimeout command in the next 
release. It should be available before the end of the week.
```

Original comment by `florentbr` on 17 Oct 2012 at 4:13
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Support**

**Comment by GoogleCodeExporter** [comment]
```
Thats damn fast! Thanks a lot!
```

Original comment by `gruberal...@gmail.com` on 19 Oct 2012 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 1.0.6.2
```

Original comment by `florentbr` on 20 Oct 2012 at 2:13
- Changed state: **Fixed**
